cli/configssh.go

```diff
@@ -106,6 +111,78 @@ func (o sshConfigOptions) equal(other sshConfigOptions) bool {
 		o.hostnameSuffix == other.hostnameSuffix
 }
 
+func (o sshConfigOptions) writeToBuffer(buf *bytes.Buffer) error {
```

Comment:
Nice refactor 👍🏻, `io.Writer` would be even more flexible but this is fine if you want to leave it.

Reply:
I originally had this, but it doesn't have `WriteString` which feels nicer to me than casting everything to []byte or fmt.Fprint().